.gitignore

```diff
@@ -61,4 +61,4 @@ bld/
 *.VC.opendb
 
 # Visual C++ Solution
-*.sln
+*.VC.db

```

Comment:
I am using x64dbg daily ! it is a very awesome tool, thank you as well for that.

Feel free to use that, I will add you in the list of contributors.